src/test/ui/suggestions/suggest-blanket-impl-local-trait.rs

```diff
@@ -0,0 +1,26 @@
+// Check what happens when we implement another for a local trait another local traits
+// the compiler should suggest also the blanket implementation. However, the suggestion
+// don't need to be present in case of a not local trait.
+// edition:2021
+use std::fmt;
+
+trait LocalTraitOne { }
+
+trait LocalTraitTwo { }
+
+impl LocalTraitTwo for LocalTraitOne {}
```

Comment:
please test `impl LocalTrait for ForeignTrait`, `impl LocalTrait for ForeignTrait + Send`, etc.